plugins/modules/zos_unarchive.py

```diff
@@ -689,7 +689,7 @@ def _create_dest_data_set(
             else:
                 rc, hlq, err = self.module.run_command("hlq")
                 hlq = hlq.replace('\n', '')
-            cmd = "mvstmphelper {0}.RESTORE".format(hlq)
```

Comment:
I noticed you removed the ZOAU helper in this part of `zos_unarchive`. Are you going to make more changes to this other module? If not, I would suggest also linking the migration of `zos_unarchive` to this PR.